### PR TITLE
Build setup for 2.1.2.1

### PR DESCRIPTION
**Pull request by adamcfraser**

### CODE REVIEW

File: manifest/2.1.2.xml [hunk 1]
```diff
@@ -70,7 +70,7 @@
 
     <project name="gomemcached" path="godeps/src/github.com/couchbase/gomemcached" remote="couchbase" revision="0da75df145308b9a4e6704d762ca9d9b77752efc"/>
 
-    <project name="sg-bucket" path="godeps/src/github.com/couchbase/sg-bucket" remote="couchbase" revision="e774e379a0452bfff199a69e053dc565918d680c"/>
+    <project name="sg-bucket" path="godeps/src/github.com/couchbase/sg-bucket" remote="couchbase" revision="release/2.1.2.1"/>
```
Comment (bbrks) [inline manifest/2.1.2.xml]: I only see a `feature/2.1.2_MP1` branch on sg-bucket. Are we awaiting a PR or something there?

**Comment by adamcfraser** [comment]
@kimtrang Can you review and let me know if this follows the approach you outlined?

File: manifest/2.1/2.1.2.xml [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    <!-- Build Scripts (required on CI servers) -->
+    <project name="build" path="cbbuild" remote="couchbase" revision="51cdfb4ae316289e072ca631cc14eb27b671cc15">
+        <annotation name="VERSION" value="2.1.2" keep="true"/>
+        <annotation name="BLD_NUM" value="@BLD_NUM@" keep="true"/>
+        <annotation name="RELEASE" value="@RELEASE@" keep="true"/>
+    </project>
+
+
+    <!-- Sync Gateway -->
+    <project name="sync_gateway" path="godeps/src/github.com/couchbase/sync_gateway" remote="couchbase" revision="release/2.1.2"/>
```
Comment (kimtrang) [inline manifest/2.1/2.1.2.xml]: @adamcfraser  Can we locked down "release/2.1.2" to a SHA?

**Review by kimtrang — APPROVED** [review]
Looks good, thanks!

**Pull request merged**